Core/src/org/sleuthkit/autopsy/centralrepository/optionspanel/EamDbSettingsDialog.java

```diff
@@ -65,18 +67,17 @@
     /**
      * This class handles displaying and rendering drop down menu for database choices in central repo.
      */
-    private class DbChoiceRenderer extends BasicComboBoxRenderer {
+    private class DbChoiceRenderer extends JLabel implements ListCellRenderer<CentralRepoDbChoice>, Serializable {
         private static final long serialVersionUID = 1L;
         
-        public Component getListCellRendererComponent(JList list, Object value,
+        public Component getListCellRendererComponent(
```

Comment:
Add @Override annotation.

Reply:
Okay, this should be added now.  Thanks.